californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -792,10 +803,22 @@ private void sendNextBlock(final Exchange exchange, final Response response, fin
 			newSzx = status.getCurrentSzx();
 		}
 		int nextNum = status.getCurrentNum() + currentSize / newSize;
-		LOGGER.debug("sending next Block1 num={}", nextNum);
+		sendBlock(exchange, response, key, status, nextNum, newSzx);
+	}
+
+	private void SendBlockAgain(final Exchange exchange, final Response response, final KeyUri key,
```

Comment:
Until now, I hope the most methods start with a lower case, `sendBlockAgain`.

Reply:
This is just a typo, I will fix it.